src/components/tables/hooks/useHideStatsColumnsSx.ts

```diff
@@ -0,0 +1,20 @@
+import { useTenantDetails } from 'context/fetcher/Tenant';
+import { useMemo } from 'react';
+import { SxProps, Theme } from '@mui/material';
+
+function useHideStatsColumnsSxSx() {
```

Comment:
Made a hook so that we didn't need to keep duplicating fetching the `hasTenants` flag.

Reply:
Also this is not around the `theme` stuff cause this is really a one off (for right now)